coderd/provisionerdserver/provisionerdserver_test.go

```diff
@@ -125,17 +127,9 @@ func TestHeartbeat(t *testing.T) {
 	require.True(t, ok, "first heartbeat not received")
```

Comment:
`testutil.RequireRecvCtx()` is your friend, paired with an overall test timeout instead of `ctx.Background()` --- written this way it's easy for bugged code to cause the test to hang for 10 minutes.